File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Features/DialogReturnValueForm.vb
Namespace CompanionKit.Controls.Form.Features
	Public Class DialogReturnValueForm


        ''' <summary>
        ''' Handles Click event of the 'Cancel' button.
        ''' Closes the Form and set Cancel dialog result for the Form.
        ''' </summary>
        ''' <param name="sender"></param>
        ''' <param name="e"></param>
        ''' <remarks></remarks>
        Private Sub mobjCancelButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjCancelButton.Click
            MyBase.DialogResult = DialogResult.Cancel
            MyBase.Close()
        End Sub

        ''' <summary>
        ''' Handles Click event of the 'Save' button.
        ''' Closes the Form and set OK dialog result for the Form.
        ''' </summary>
        Private Sub mobjSaveButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjSaveButton.Click
            MyBase.DialogResult = WebGUI.Forms.DialogResult.OK
            MyBase.Close()
        End Sub

        ''' <summary>
        ''' Gets or sets text value of the TextBox that represents user email.
        ''' </summary>
		Public Property Email() As String
			Get
                Return Me.mobjEmailTextBox.Text
			End Get
			Set(ByVal value As String)
                Me.mobjEmailTextBox.Text = value
			End Set
		End Property

        ''' <summary>
        ''' Gets or sets text value of the TextBox that represents user phone.
        ''' </summary>
		Public Property Phone() As String
			Get
                Return Me.mobjPhoneTextBox.Text
			End Get
			Set(ByVal value As String)
                Me.mobjPhoneTextBox.Text = value
			End Set
		End Property

        ''' <summary>
        ''' Gets or sets text value of the TextBox that represents user first name.
        ''' </summary>
        ''' <value></value>
        ''' <returns></returns>
        ''' <remarks></remarks>
		Public Property UserFirstName() As String
			Get
                Return Me.mobjUserFirstNameTextBox.Text
			End Get
			Set(ByVal value As String)
                Me.mobjUserFirstNameTextBox.Text = value
			End Set
		End Property

        ''' <summary>
        ''' Gets or sets user full name.
        ''' </summary>
		Public ReadOnly Property UserFullName() As String
			Get
                Return String.Format("{0} {1}", Me.mobjUserFirstNameTextBox.Text, Me.mobjUserLastNameTextBox.Text)
			End Get
		End Property

        ''' <summary>
        ''' Gets or sets text value of the TextBox that represents user last name.
        ''' </summary>
		Public Property UserLastName() As String
			Get
                Return Me.mobjUserLastNameTextBox.Text
			End Get
			Set(ByVal value As String)
                Me.mobjUserLastNameTextBox.Text = value
			End Set
		End Property


	End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Features/DialogReturnValueForm.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;

using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.Form.Features
{
    public partial class DialogReturnValueForm : global::Gizmox.WebGUI.Forms.Form
    {
        public DialogReturnValueForm()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Click event of the 'Save' button.
        /// Closes the Form and set OK dialog result for the Form.
        /// </summary>
        private void mobjSaveButton_Click(object sender, EventArgs e)
        {
            this.DialogResult = DialogResult.OK;
            this.Close();
        }

        /// <summary>
        /// Handles Click event of the 'Cancel' button.
        /// Closes the Form and set Cancel dialog result for the Form.
        /// </summary>
        private void mobjCancelButton_Click(object sender, EventArgs e)
        {
            this.DialogResult = DialogResult.Cancel;
            this.Close();
        }

        /// <summary>
        /// Gets or sets text value of the TextBox that represents user first name.
        /// </summary>
        public String UserFirstName
        {
            get
            {
                return this.mobjUserFirstNameTextBox.Text;
            }

            set
            {
                this.mobjUserFirstNameTextBox.Text = value;
            }
        }

        /// <summary>
        /// Gets or sets text value of the TextBox that represents user last name.
        /// </summary>
        public String UserLastName
        {
            get
            {
                return this.mobjUserLastNameTextBox.Text;
            }

            set
            {
                this.mobjUserLastNameTextBox.Text = value;
            }
        }

        /// <summary>
        /// Gets or sets user full name.
        /// </summary>
        public String UserFullName
        {
            get
            {
                return string.Format("{0} {1}", this.mobjUserFirstNameTextBox.Text, this.mobjUserLastNameTextBox.Text);
            }
        }

        /// <summary>
        /// Gets or sets text value of the TextBox that represents user phone.
        /// </summary>
        public String Phone
        {
            get
            {
                return this.mobjPhoneTextBox.Text;
            }

            set
            {
                this.mobjPhoneTextBox.Text = value;
            }
        }

        /// <summary>
        /// Gets or sets text value of the TextBox that represents user email.
        /// </summary>
        public String Email
        {
            get
            {
                return this.mobjEmailTextBox.Text;
            }

            set
            {
                this.mobjEmailTextBox.Text = value;
            }
        }

    }
}